src/main/resources/static/api.yml

```diff
@@ -2389,6 +2389,32 @@ paths:
           $ref: '#/components/responses/403Response'
         500:
           $ref: '#/components/responses/500Response'
+  /placement-applications/{id}:
+    get:
+      tags:
+        - Placement applications
+      summary: Creates an application for a placement request
```

Comment:
think this should be `Retrieves an application for a placement request`

Reply:
Done!